docs/troubleshooting.rst

```diff
@@ -24,8 +24,37 @@ retaining the database::
   git submodule sync --recursive && git submodule update --init --recursive
   docker-compose pull
   docker-compose build --pull
-  docker-compose up
+  docker-compose up -d
```

Comment:
I think its better to not add ``-d`` in developer environment until its necessary. So it will be easy to see the logs